### ISSUE TITLE
Combining Gnome's Workrave extension does not work correctly in Wayland

### ISSUE THREAD

**Issue by tukanos** [rcaelers/workrave#527]
**Describe the bug**
Running Workrave extesion with autostart Workrave does not work correctly.  The checkbox "Force the use of X11 on Wayland" is checked in both cases.

- When running Workrave extension - I end up with working applet, but not Workrave itself.  
- If I disable the extension the Workraved itself works but I don't get the applet.  

**To Reproduce**
1. `cp` the workrave.desktop into `~/.config/autostart`
2. Start gnome's extensions and enable the workrave extension
3. After the login the extension is started first and so the autostarted workrave throws error that workrave is alrady started  

**Expected behavior**
There are two ways to approach it:
a. The workrave extension to add the env variable `GDK_BACKEND=x11` as the `.desktop` file has
b. Enable the extension only after the autostarted Workrave

**Workaround**
Enable the Workrave extension with delay on Gnome start:
` gnome-extensions enable workrave@workrave.org`
At shutdown disable the extension with
`gnome-extensions disable workrave@workrave.org`

** Linux (please complete the following information in case you encountered the bug on Linux):**
 - Distribution: Debian 
 - Distribution Version: Debian Testing
 - Desktop Environment: Gnome 45
 - Workrave Version: 1.10.52-1

**Additional context**
I would like for Workrave to work as on Xorg server on Wayland.  

- If I start Workrave via `.desktop` it works correctly (as it is started as `Exec=/usr/bin/env GDK_BACKEND=x11 workrave` but without the nice applet (via extensions) on top.   If I remove the autostart and switch on the Workrave extension the applet is there but the break is not over the whole screen and I can work with

The details of installed debian packages:
```
apt info workrave-gnome-flashback
Package: workrave-gnome-flashback
Version: 1.10.52-1
Priority: optional
Section: gnome
Source: workrave
Maintainer: Francois Marier <francois@debian.org>
Installed-Size: 68.6 kB
Depends: gnome-panel, workrave (>= 1.10.52-1), libc6 (>= 2.4), libgdk-pixbuf-2.0-0 (>= 2.22.0), libglib2.0-0 (>= 2.38.0), libgnome-panel3 (>= 3.37.1), libgtk-3-0 (>= 3.0.0)
Homepage: https://workrave.org/
Tag: uitoolkit::gtk
Download-Size: 17.8 kB
APT-Manual-Installed: yes
APT-Sources: https://deb.debian.org/debian trixie/main amd64 Packages
Description: Repetitive Strain Injury prevention tool (GNOME panel applet)
 Workrave is a program that assists in the recovery and prevention of
 Repetitive Strain Injury (RSI). The program frequently alerts you to
 take micro-pauses, rest breaks and restricts you to your daily limit.
 .
 It includes a system tray applet that works with GNOME and KDE
 and has network capabilities to monitor your activity even if
 switching back and forth between different computers is part of your
 job.
 .
 Workrave offers many more configuration options than other similar
 tools.
 .
 This package includes a Workrave applet for GNOME Flashback.
 
-----------------------------------------------------------------------

apt info workrave-gnome
Package: workrave-gnome
Version: 1.10.52-1
Priority: optional
Section: gnome
Source: workrave
Maintainer: Francois Marier <francois@debian.org>
Installed-Size: 51.2 kB
Depends: gnome-shell (<< 45~), workrave (>= 1.10.52-1)
Homepage: https://workrave.org/
Download-Size: 14.2 kB
APT-Manual-Installed: yes
APT-Sources: https://deb.debian.org/debian trixie/main amd64 Packages
Description: Repetitive Strain Injury prevention tool (GNOME integration)
 Workrave is a program that assists in the recovery and prevention of
 Repetitive Strain Injury (RSI). The program frequently alerts you to
 take micro-pauses, rest breaks and restricts you to your daily limit.
 .
 It includes a system tray applet that works with GNOME and KDE
 and has network capabilities to monitor your activity even if
 switching back and forth between different computers is part of your
 job.
 .
 Workrave offers many more configuration options than other similar
 tools.
 .
 This package includes a Workrave GNOME Shell extension.
 
-----------------------------------------------------------------------

apt info workrave
Package: workrave
Version: 1.10.52-1
Priority: optional
Section: gnome
Maintainer: Francois Marier <francois@debian.org>
Installed-Size: 2,480 kB
Depends: workrave-data (= 1.10.52-1), libatkmm-1.6-1v5 (>= 2.28.3), libc6 (>= 2.34), libcairo2 (>= 1.10.0), libcairomm-1.0-1v5 (>= 1.14.4), libdbusmenu-glib4 (>= 0.4.2), libgcc-s1 (>= 3.0), libgdk-pixbuf-2.0-0 (>= 2.22.0), libgdome2-0 (>= 0.8.1), libglib2.0-0 (>= 2.40.0), libglibmm-2.4-1v5 (>= 2.66.6), libgstreamer1.0-0 (>= 1.0.0), libgtk-3-0 (>= 3.21.4), libgtk-4-1 (>= 4.0.0), libgtkmm-3.0-1v5 (>= 3.24.8), libpango-1.0-0 (>= 1.14.0), libpangocairo-1.0-0 (>= 1.14.0), libpangomm-1.4-1v5 (>= 2.46.3), libsigc++-2.0-0v5 (>= 2.8.0), libstdc++6 (>= 13.1), libx11-6, libxss1, libxtst6
Suggests: workrave-ayatana, workrave-cinnamon, workrave-gnome, workrave-gnome-flashback, workrave-mate, workrave-xfce4
Breaks: workrave-data (<< 1.10.51.1-4)
Homepage: https://workrave.org/
Tag: accessibility::TODO, hardware::input, implemented-in::c++,
 interface::graphical, interface::x11, role::program, scope::utility,
 suite::gnome, uitoolkit::gtk, use::monitor, x11::applet,
 x11::application
Download-Size: 549 kB
APT-Manual-Installed: yes
APT-Sources: https://deb.debian.org/debian trixie/main amd64 Packages
Description: Repetitive Strain Injury prevention tool
 Workrave is a program that assists in the recovery and prevention of
 Repetitive Strain Injury (RSI). The program frequently alerts you to
 take micro-pauses, rest breaks and restricts you to your daily limit.
 .
 It includes a system tray applet that works with GNOME and KDE
 and has network capabilities to monitor your activity even if
 switching back and forth between different computers is part of your
 job.
 .
 Workrave offers many more configuration options than other similar
 tools.
```

**Comment by tukanos** [comment]
I did some digging and I have found out that the Workrave extension (Gnome) is for Gnome 44, for 45 You have an update.

**Comment by Chealer** [comment]
Does this persist in Debian 13 (or current testing)?

**Comment by tukanos** [comment]
> Does this persist in Debian 13 (or current testing)?

It was fine on Gnome 48, but it again has issues on Gnome 49.

**Comment by Chealer** [comment]
Thank you
Which exact distribution are you using?

**Comment by tukanos** [comment]
> Thank you Which exact distribution are you using?

My pleasure.  I'm using pure Debian Testing (Forky).

**Comment by Chealer** [comment]
Thank you
Debian testing is using GNOME 49 and Workrave 1.10.54, which is implicitly advertised as supporting GNOME 49. Can you confirm you are now using workrave 1.10.54-2?

**Comment by tukanos** [comment]
> Thank you Debian testing is using GNOME 49 and Workrave 1.10.54, which is implicitly advertised as supporting GNOME 49. Can you confirm you are now using workrave 1.10.54-2?

Yes, I'm using that version.  Here is my apt output:

```
$ apt search workrave | rg installed

workrave/testing,now 1.10.54-2 amd64 [installed]
workrave-data/testing,testing,now 1.10.54-2 all [installed,automatic]
workrave-gnome/testing,testing,now 1.10.54-2 all [installed]
```

**Comment by Chealer** [comment]
Thanks
So, if I understand correctly, you are no longer using workrave-gnome-flashback?
When you talk about breaks, are you talking about both micro-breaks are rest breaks?

**Comment by tukanos** [comment]
> Thanks So, if I understand correctly, you are no longer using workrave-gnome-flashback? When you talk about breaks, are you talking about both micro-breaks are rest breaks?

a) I'm no longer using it.  The whole Gnome 49 upgrade did not end up well in my eyes

b) Yes, I'm talking about both.

**Comment by Chealer** [comment]
Thank you Patrik

The extension you refer to is the extension of GNOME intended to let you use [the applet in GNOME](https://workrave.org/docs/applets/gnome/), right?

> **Expected behavior** The workrave extension to should add the env variable `GDK_BACKEND=x11` as the `.desktop` file has.

The _Expected behavior_ section should be phrased in functional terms. For example, if you report that your car's window control button has broken, that section does not have to say that the window's engine should power up, but should (basically) say that the window should move.

**Comment by tukanos** [comment]
> Thank you Patrik
> 
> The extension you refer to is the extension of GNOME intended to let you use [the applet in GNOME](https://workrave.org/docs/applets/gnome/), right?

You are welcome. Yes.

> > **Expected behavior** The workrave extension to should add the env variable `GDK_BACKEND=x11` as the `.desktop` file has.
> 
> The _Expected behavior_ section should be phrased in functional terms. For example, if you report that your car's window control button has broken, that section does not have to say that the window's engine should power up, but should (basically) say that the window should move.

Of course, my idea of functional is that Workrave will cover whole screen and have a one window in the middle with countdown.  This is how it worked in X11.  

I've also found out that the Gnome 49 was somehow damaged by the upgrade from Gnome 48.  I'll have to find the issue and then I'll test it properly again.

**Comment by Chealer** [comment]
Thank you Patrik

> > > **Expected behavior** The workrave extension to should add the env variable `GDK_BACKEND=x11` as the `.desktop` file has.
> > 
> > 
> > The _Expected behavior_ section should be phrased in functional terms. For example, if you report that your car's window control button has broken, that section does not have to say that the window's engine should power up, but should (basically) say that the window should move.
> 
> Of course, my idea of functional is that Workrave will cover whole screen and have a one window in the middle with countdown. This is how it worked in X11.

~~Hum, I never used Workrave under GNOME, but I used it under X11 for years and never witnessed Workrave covering the whole screen. Can you show a screenshot of that?~~

**Comment by JVC-MSF** [comment]
I'm having the same issue with Debian 13 (trixie) using Gnome with Wayland.

> workrave-gnome/stable,now 1.10.53-2 all [instalado]
> workrave/stable,now 1.10.53-2 amd64 [instalado, automático]
> gnome-shell/stable,now 48.7-0+deb13u1 amd64 [instalado, automático]

I'm using a laptop with two external monitors, that is three screens.
When a break or a micro-break triggers, the blocking "thing" (the black square with the dialog in the middle to skip or postpone) has the size of the smaller physical screen (the laptop one) and I can see the back on the sides, and all three blocking are in the main scree.

The keyboard is correctly blocked.

Hope this info helps.

**Comment by tukanos** [comment]
> Thank you Patrik
> 
> > > > **Expected behavior** The workrave extension to should add the env variable `GDK_BACKEND=x11` as the `.desktop` file has.
> > > 
> > > 
> > > The _Expected behavior_ section should be phrased in functional terms. For example, if you report that your car's window control button has broken, that section does not have to say that the window's engine should power up, but should (basically) say that the window should move.
> > 
> > 
> > Of course, my idea of functional is that Workrave will cover whole screen and have a one window in the middle with countdown. This is how it worked in X11.
> 
> Hum, I never used Workrave under GNOME, but I used it under X11 for years and never witnessed Workrave covering the whole screen. Can you show a screenshot of that?

Ah I'm sorry for belated answer, things got really busy here.  Here is screenshot of my monitor - notice the top and bottom is still shown even when Workrave is running:

<img width="1024" height="474" alt="Image" src="https://github.com/user-attachments/assets/27f65d38-6663-40fe-9dc8-1577f6a64ab9" />

**Comment by Chealer** [comment]
Thank you @JVC-MSF and Patrik

@tukanos: The image is unfortunately quite blurry. Is it possible to provide 2 screenshots of 1. during the break and 2. right after?

Also, is there a reason why you renamed this ticket removing "Wayland" from the title?

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>

**Comment by tukanos** [comment]
> Thank you [@JVC-MSF](https://github.com/JVC-MSF) and Patrik
> 
> [@tukanos](https://github.com/tukanos): The image is unfortunately quite blurry. Is it possible to provide 2 screenshots of 1. during the break and 2. right after?
> 
> Also, is there a reason why you renamed this ticket removing "Wayland" from the title?
> License

Yes, it is bit blurry.  Hopefully these screenshots are better:

1.

![Image](https://github.com/user-attachments/assets/029a8953-64a9-4b00-9d14-d2e5a9cb0af1)

2.

![Image](https://github.com/user-attachments/assets/d698f1f1-97e4-4f0e-ba22-2df22c3eed06)

Why did I remove Wayland?  I honestly don't remember, probably I thought it was a good idea back then.  I have added it it back.

**Comment by Chealer** [comment]
Thank you Patrik
I believe I now understand the difference between the behaviors you expect and get.

@JVC-MSF: Are you using the GNOME applet? And if so, does disabling it solve the problem?

**Comment by JVC-MSF** [comment]
Hi @Chealer 
I have tried with and without the applet and the behavior is the same. 
I have also tried to "Force the use of X11 in Wayland" to no avail.

If I can do any specific test to help clarify this, please do not hesitate.

**Comment by Chealer** [comment]
Thank you @JVC-MSF
This ticket is about the GNOME applet, so I doubt your issue is the same. Can you provide a screenshot of your smallest screen during a break (ideally when the applet is enabled)?

**Comment by JVC-MSF** [comment]
Ok.

 I can't take a screenshot because the screen is locked, I thin, I have never tried 😃.

If I can't manage, I will send a photo.

**Comment by JVC-MSF** [comment]
Here you have photos taken to my three screens. I took them in "overview mode" (don0t know how to call it.

Notice that the black squares are all in the main screen instead of in the three.

Hope this helps.

![Image](https://github.com/user-attachments/assets/755d0dff-a4d0-456e-9585-c65b3c15fd81)
![Image](https://github.com/user-attachments/assets/ce090f57-a0ce-4b4b-b426-79c5bd89ff4a)
![Image](https://github.com/user-attachments/assets/95d0c43e-40f8-4b35-9fb2-37d788d41b7b)

**Comment by JVC-MSF** [comment]
I just noticed that the black squares appear in the screen where the mouse is.

**Comment by Chealer** [comment]
@JVC-MSF: I am unable to read the Workrave windows in these pictures, but thank you. Workrave has an option to lock input devices. You can disable it in the User interface section of Preferences to facilitate taking a screenshot. In fact, could you show which value you currently have there?

I struggle to see your issue and Patrik's as a single one. If you use a single screen, do you get the same behavior as Patrik?

**Comment by JVC-MSF** [comment]
Today I'm not going to be able. Tomorrow I will look at it.

**Comment by JVC-MSF** [comment]
Here you have them. A preferences capture, lock screen capture (the other two screes still show their contents, an a capture where you can see that the three screen blockers are all of them in the main screen.
Hope this is more helpful.

<img width="2400" height="1350" alt="Image" src="https://github.com/user-attachments/assets/646bf00e-616f-4f43-b24a-18d86dd1175c" />
<img width="2400" height="1350" alt="Image" src="https://github.com/user-attachments/assets/e393035a-4b54-450a-b203-0318c0955e23" />
<img width="2400" height="1350" alt="Image" src="https://github.com/user-attachments/assets/cde047ec-b9f1-479b-b59f-efc99cb80501" />

**Comment by Chealer** [comment]
Thank you very much @JVC-MSF, that is much clearer, but I still do not fully understand. If you have a single active screen, do you get the same behavior as Patrik?

Your block mode ([`block_mode`](https://github.com/rcaelers/workrave/blob/01562061eb7901738e83d89dc25a3dca3f1e4ce1/ui/app/toolkits/gtkmm/preferences/GeneralPreferencePanel.cc#L79)) is set to [“Bloquear teclado, ratón y pantalla”, which means "Block input and screen"](https://github.com/rcaelers/workrave/blob/01562061eb7901738e83d89dc25a3dca3f1e4ce1/po/es.po#L94). Is it intentional that you block the screen during breaks?

**Comment by JVC-MSF** [comment]
I never work with one monitor only. The laptop scree is too small 😄. I will try to do a testing round today with the extra screens unplugged.

On the blocking mode. I think it is the default. Anyway, I like it like that so I get pushed away from the computer for some minutes, that feels good for my health. I will try with different settings to check if it makes a difference.

**Comment by Chealer** [comment]
@JVC-MSF: Indeed, laptop screens are small; I use a single screen at any time, but it is my monitor’s. I keep the laptop’s lid closed, so only my monitor’s screen is active.
Thank you, you seem to be right that screens are blocked by default.

@tukanos: Is your block mode set to block screens, or what?

**Comment by JVC-MSF** [comment]
As I said, I prefer the block screen mode, this ensures that I will move away and rest my eyes too. So, I will do my tests in this mode, unless you tell me to do it in a different configuration for testing shake.

I have been running some testing without any additional monitor. In that case, the only visual issue is that the top bar is still visible and my expectation is that it will be also hidden.

**Comment by Chealer** [comment]
Thanks again @JVC-MSF 

I tested the block screen mode on Windows 11, and everything (but the desktop background) is hidden. So I guess what you report is a defect.

@JVC-MSF: Does the strange display shown in your second screenshot from 2026-02-13 occur if your block mode is not set to block screens?

**Comment by JVC-MSF** [comment]
Hi @Chealer 

If the mode is non blocking, I get three dialogs informing on the break in the same screen. Si, I will say yes, it is still strange.

**Comment by Chealer** [comment]
Thanks again @JVC-MSF 

I do not see a bottom bar in any of your screenshots; if I understand correctly, that is because you have no bottom bar in regular use, right?
Also, would you mind sharing how you managed to take the (good quality) first 2 screenshots in your 2026-02-13 comment given that your input is blocked during breaks?

**Comment by JVC-MSF** [comment]
Hi @Chealer 

Yes, I don't use a bottom bar. 

And how I managed to take the screenshots. Well, because my only my keyboard is blocked, but not my mouse. With the mouse alone I can take a screen capture. I only realized when I was trying to do the screen capture.

I assume that it is because the screen is not properly "hidden" behind the black square.

**Comment by Chealer** [comment]
Ah, indeed, I see, the occasional helpful bug😏
Thanks again Javier
In that case, is pointer blocking only failing in the top bar, or are you also able to click in windows (other than Workrave’s)?

----

I will try to recapitulate where we are so far.
2 symptoms were reported in this ticket:
1. The horizontal (top/bottom) bar(s) showing during breaks (both micro-breaks and rest breaks) in GNOME when using Wayland (not X11)
2. A display which I find strange and hard to describe―probably because I never used GNOME―experienced by Javier during breaks when using multiple screens on Wayland, as he showed in the second screenshot(s) of his 2026-02-09 15:25 and 2026-02-13 comments.

According to Javier, #1 only occurs when input and display are blocked. @tukanos has yet to confirm that.

Assuming that this only happens when blocking, it is more difficult to capture "real" screenshots of the problem, but it may be possible thanks to the mouse.

----

It remains unclear to me whether these 2 symptoms are caused by the same bug or distinct. I unfortunately do not have GNOME and cannot help determine whether the applet is at fault.
@JVC-MSF: Patrik’s report implies rather clearly that #1 does not occur without the applet, but according to your 2026-02-06 comment the applet is irrelevant. Are you confident about that? If you disable the extension, close your session, and open a new session, does #1 occur (when the applet has not run at all during the session)?

**Comment by JVC-MSF** [comment]
Hi @Chealer 

On point 1.  I can't use X11 anymore in my Debian 13 + GNOME. If I try to force to go X11, I must go to GNOME Classic.

About your point 2. I think I have confused you. I'm very sorry. "the strange and hard to describe" screen is a regular Gnome thing. When you move the mouse pointer  to the upper left corner, you get a view where you see all the windows you have in the workarea, in reduced size.

I was showing it because there was clear that workrave was painting a dialog and "coves black square" for each window but painting all of them in the monitor that has the mouse at the moment of triggering the break, or micro break.

I will disable the extension an will work tomorrow all the day without it for testing.

Thanks for dedicating your time to that lovely tool :)

Cheers,
Javi

**Comment by Chealer** [comment]
Ah, thank you very much Javier, that clarifies a lot.
Javier seems to be talking about GNOME’s "Hot corner" feature, which reveals the _Activities_ view ([that page](https://www.debugpoint.com/gnome-40-workspace-navigation/) shows something similar).
@JVC-MSF: If I understand correctly, the top fifth of your screenshots show you had 3 workspaces on 2026-02-09, and 4 on 2026-02-13 (you can answer later this week after your tests)?

**Comment by JVC-MSF** [comment]
Hi @Chealer 

Yes this is what I was talking about. Sorry for not using the right terms. 

And yes, I heavily use workspaces. In gnome they are dynamic and very helpful way to organize my work 😄 

I have been running workrave without the extension for a full day and results where very similar. But surprisingly, the window sized was smaller. Here you have the screen captures:

<img width="2400" height="1350" alt="Image" src="https://github.com/user-attachments/assets/7d89e0b1-6a46-49f3-9791-6b5d47d60f53" />

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/473a765b-d620-4732-9a95-5bcda71c92c1" />

They are from the central and left screens.

Also to mention that, in GNOME, workrave without extension is not very pleasant to use due to the lack of icon. You have the pauses happening but no way to know when they are going to happen and there's no access to the preferences.

**Comment by Chealer** [comment]
Thank you once again Javier

So, if I understand correctly, the relevant part of your second screenshot from 2026-02-13 is the central ¾. Is it correct that this center shows you had 6 windows open on your 3 active screens during your rest break, 3 useful windows and 3 temporary windows by Workrave?

> Also to mention that, in GNOME, workrave without extension is not very pleasant to use due to the lack of icon. You have the pauses happening but no way to know when they are going to happen and there's no access to the preferences.

Hum. As I said, I never used Workrave on GNOME, but I cannot help but being a little skeptical that this situation would be normal. Do you know if that is a known issue?

**Comment by Chealer** [comment]
@tukanos: Javier tested that on his system, the same symptoms occur even in a session where the applet was never enabled.
How confident are you that Workrave leaves the horizontal bars displayed during breaks only if the applet is/was started?

**Comment by tukanos** [comment]
> [@tukanos](https://github.com/tukanos): Javier tested that on his system, the same symptoms occur even in a session where the applet was never enabled. How confident are you that Workrave leaves the horizontal bars displayed during breaks only if the applet is/was started?

That could be the case.  My understanding is that the applet is there helping Workrave to work correctly under Wayland.  Is my assumption incorrect?

The only way to know for sure to debug it and see the reason why this could be happening.

**Comment by Chealer** [comment]
@tukanos wrote:
> My understanding is that the applet is there helping Workrave to work correctly under Wayland. Is my assumption incorrect?

[The applet’s documentation](https://workrave.org/docs/applets/gnome/) makes no mention of working correctly, only of showing all timer values.

Unless you are confident that this is caused by the extension, I recommend retitling this to something like:
> GNOME taskbar(s) stay visible during breaks on Wayland when screen blocking is enabled

**Comment by JVC-MSF** [comment]
Sorry. I have been away for a week.

> Hum. As I said, I never used Workrave on GNOME, but I cannot help but being a little skeptical that this situation would be normal. Do you know if that is a known issue?

I don't know. I have been using workave for 5 years or so in KDE+X11 at work. A couple of months ago I move to GNOME+Wayland (which I personally prefer) and I started to have the issues that bring me to this GH issue.

As a user, I find that the current behavior is unpleasant and I will consider it a bug. But the lack of proper screen blocking with the black layer, as I had in KDE, is the biggest issue right now.

**Comment by Chealer** [comment]
Thanks Patrik, and thanks again Javier

I found that the lack of tray icon is a known issue with GNOME, tracked in ticket #265.

> From GNOME 3.26, we are therefore planning not to show status icons in GNOME Shell by default. […] We also feel that the consequences of the change won’t be as dramatic as they would have been in the past.

😆

**Comment by JVC-MSF** [comment]
😄 
If I have the GNOME Shell extension  is OK for me. So, not a big issue.

But will be nice to have the "black blocking thing" in  the right place.

**Comment by tukanos** [comment]
> Thanks Patrik, and thanks again Javier
> 
> I found that the lack of tray icon is a known issue with GNOME, tracked in ticket [#265](https://github.com/rcaelers/workrave/issues/265).
> 
> > From GNOME 3.26, we are therefore planning not to show status icons in GNOME Shell by default. […] We also feel that the consequences of the change won’t be as dramatic as they would have been in the past.
> 
> 😆

Happy to help.  I do have tray icon in Gnome, as you can see from my screenshots, you need to have an extension for that.  In my case I'm using `AppIndicator and KStatusNotfierItem support` extension.
